tests/GeneratedTests.hs

```diff
@@ -78,8 +78,8 @@ testClientGeneration = testCase "client generation" $ do
       serverExitCode <- liftIO (wait serverExitCodeA)
       clientExitCode <- liftIO (wait clientExitCodeA)
 
-      serverExitCode @?= ExitSuccess
-      clientExitCode @?= ExitSuccess
+      assertEqual "Server exit code" serverExitCode ExitSuccess
```

Comment:
I got confused when I got failures so I prefer being more verbose here to make it clear what failed.